vendor/github.com/src-d/code-annotation/src/pages/Experiment.less

```diff
@@ -0,0 +1,74 @@
+@import '../../node_modules/bootstrap/less/variables.less';
```

Comment:
I'm not sure why we see this file here. It shouldn't be here.

Reply:
Don't know. I started working when I was supposed to, and with the latest version.

Reply:
you need to fix it.

Reply:
I'll do it, but I thought that we had learned from the Annotation PR - where we had conflicts and I had to solve them. We're still figuring out the best way to team up and be productive, but I think that the "time-stamped" version for me to work on, should convey the fact that I need time to do the Visual Design and not making me think of app file structure and changes in the process. What would you suggest, after what I've mentioned?

Reply:
you changed the wrong file. It should be 
```
src/pages/Experiment.less
```

not
```
vendor/github.com/src-d/code-annotation/src/pages/Experiment.less
```

Reply:
or there is something wrong with your local environment. Actually `vendor/github.com/src-d/code-annotation` shouldn't exist at all.